docs/faq.md

```diff
@@ -67,16 +58,16 @@ hide:
       * 16x full-resolution (10-bit, ext-limits, all half rate) with 1x 2-position channel, AUX1
       * 12x full-resolution (10-bit, ext-limits) with 1x 2-position channel, AUX1, channels AUX2-9 run at half rate.
         
-      See [Switch Modes](software/switch-config.md) for more details on switch modes and how they work
+      See [Switch Modes](https://www.expresslrs.org/software/switch-config.md) for more details on switch modes and how they work.
 
-## <span class="custom-heading" data-id="10">Is my binding phrase a secret?</span>
+## <span class="custom-heading" data-id="9">Is my binding phrase a secret?</span>
 
 ??? faq "Is my binding phrase a secret?"
-    No, just like what channel your VTX is on is not a secret. The binding phrase is not security, it is anti-collision. If everyone kept their VTX channel a secret, the chances of you blasting someone out of the sky accidentally is pretty high. To provide the best chance of not interfering with other pilots and them not interfering with you, be sure you're not using the same dumb bind phrase as someone else. Express your style with a hilarious or saucy bind phrase.
+    No, just like what channel your VTX is on is not a secret. The binding phrase is not used for security, it is used for anti-collision. If everyone kept their VTX channel a secret, the chances of you blasting someone out of the sky accidentally is pretty high. To provide the best chance of not interfering with other pilots and them not interferingwith you, be sure you're not using the same dumb bind phrase as someone else. Express your style with a hilarious or saucy bind phrase.
```

Comment:
The binding phrase is not used for security. It is for anti-collision.

interferingwith - space needed

Reply:
Updated.